src/js/components/AppConfigJSONEditorComponent.jsx

```diff
@@ -0,0 +1,63 @@
+import AceEditor from "react-ace";
+import "brace/mode/json";
+import "brace/theme/monokai";
+import React from "react/addons";
+import Util from "../helpers/Util";
+
+var AppConfigJSONEditorComponent = React.createClass({
+  displayName: "AppConfigJSONEditorComponent",
+
+  propTypes: {
+    app: React.PropTypes.object,
+    onChange: React.PropTypes.func,
+    onError: React.PropTypes.func
+  },
+
+  getDefaultProps: function () {
+    return {
+      app: {id: ""},
```

Comment:
Sure, I think it could make sense to have some 'shown by default' fields which are not necessarily required.